### ISSUE TITLE
Add new parameter to remove proteins

### ISSUE THREAD

**Issue by veitveit** [veitveit/PhosFake#13]
To better reproduce cell-type specific protein expression, randomly remove a (different) set of proteins from each sample type.

**Comment by mlocardpaulet** [comment]
thinking about it... 
It may work this way, but since we keep track of regulations in the ground truth, either we should remove the same proteins in all the samples, either we should keep track of what proteins are present in a condition and not in the other one. Right?

**Comment by veitveit** [comment]
Right, because the missingness would enforce a certain ground truth.

When removing proteins from all samples, then this would definitely be easier. Maybe just discard a certain percentage directly after reading the fasta?

**Comment by veitveit** [comment]
Should be done

**Comment by mlocardpaulet** [comment]
How is this new parameter called?

**Comment by mlocardpaulet** [comment]
Found it! `PercExpressedProt`
